cmd/cloud/server.go

```diff
@@ -126,7 +117,7 @@ var serverCmd = &cobra.Command{
 			"debug":                           debug,
 		}).Info("Starting Mattermost Provisioning Server")
 
-		deprecationWarnings(logger)
+		deprecationWarnings(logger, command)
```

Comment:
Interesting, where are you seeing  this? A pointer to command needs to be passed to `deprecationWarnings` so that we can call `Flags()` and check for deprecated params.

Reply:
Sorry, I read this signature too quickly. We are good to go.